lib/errors.js

```diff
@@ -51,14 +53,18 @@ class CliError extends Error {
     toString(verbose) {
         let output = `${chalk.yellow('Message:')} ${this.message}\n`;
 
-        if (verbose) {
-            output += `${chalk.yellow('Stack:')} ${this.stack}\n`;
-        }
-
         if (this.options.help) {
             output += `${chalk.gray('Help:')} ${this.options.help}\n`;
         }
 
+        if (this.options.context.suggestion) {
+            output += `${chalk.green('Suggestion:')} ${this.options.context.suggestion}\n`;
+        }
+
+        if (verbose) {
```

Comment:
reordered. it looks very unreadable if we first print the stack and then a small line with `Help:`.